profiles/killbill/src/test/java/org/killbill/billing/jaxrs/TestBuildResponse.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2010-2013 Ning, Inc.
+ * Copyright 2014 Groupon, Inc
+ * Copyright 2014 The Billing Project, LLC
+ *
```

Comment:
Nit: You don't need to include `Ning` or `Groupon` for any of the new files you create. Also if you care about adding the name of your consulting company this is fine too.